search/releases/6.0/index.adoc

```diff
@@ -52,36 +38,43 @@ A lot of APIs have been changed, for multiple reasons.
 
 The API types consistently use the `Search` prefix: no more mixing `FullText` with `Search` or simply no prefix.
 
-Additionally, the `SearchQuery` type (previously `FullTextQuery`) now defines its own methods
+Additionally, the `SearchQuery` type (previously `FullTextQuery`)
+now link:{hsearch-doc-url-prefix}#search-dsl-query-fetching-results[defines its own methods]
 instead of extending JPA's `TypeQuery`, allowing for an API that makes more sense considering that an index,
```

Comment:
I know this line is not part of the current changes. But I don't understand "allowing" what "to do" what exactly.

Reply:
> allow for [smth]: make [sth] possible.
> Ex.: We must make room to allow for expansion.

Here the new `SearchQuery` type, which defines its own methods, makes the new, more sensical API possible.

Reply:
>  allowing for an API that makes more sense considering that an index, ...

Maybe you could rewrite it as:

`highlighting the fact that it targets an index and not a database, and making it a friendlier API`

Just my 2c

Reply:
Alright. Done! Thanks.